src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3341,6 +3363,25 @@ private int GetOrCreatePlaceholderSlot(object identifier, TypeWithAnnotations ty
             return GetOrCreateSlot(placeholder, forceSlotEvenIfEmpty: true);
         }
 
+        private bool TryGetPlaceholderSlot(BoundExpression node, out int slot)
+        {
+            if (_placeholderLocalsOpt?.TryGetValue(node, out var placeholderLocal) == true)
```

Comment:
>_placeholderLocalsOpt [](http://example.com/codeflow?start=16&length=21)

I think the implementation can be simpler if we change ```getInterpolatedStringHandlerPlaceholderSlot``` to always add an entry into ```_interpolatedStringArgumentPlaceholderAliases```. This way we wouldn't need to look in two places.